chainer/functions/loss/black_out.py

```diff
@@ -27,14 +27,33 @@ def black_out(x, t, W, samples):
        p(y) = \\frac{\\exp(W_y^\\top x)}{
        \\sum_{s \\in samples} \\exp(W_s^\\top x)}.
 
+    The output is a variable whose value depends on the value of
+    the option ``reduce``. If it is ``'samplewise'``, it holds the
```

Comment:
Because I first thought it is more meaningful than `'no'`. But I come to think `'no'` is also OK and further, we can align it with other functions. So, I will change it to `'no'`.